routes/web.php

```diff
@@ -15,8 +16,35 @@
 
 Route::get('/', function () {
     return view('welcome');
+})->name('welcome');
+
+Route::middleware(['auth'])->group(function () {
+
+    Route::get('movies/create', [MovieController::class, 'create'])->name('movies.create');
+
+    Route::get('movies/{movie}/edit', [MovieController::class, 'edit'])->name('movies.edit');
+
+    Route::delete('movies/{movie}', [MovieController::class, 'destroy'])->name('movies.destroy');
 });
 
+Route::get('movies', [MovieController::class, 'index'])->name('movies.index');
+
+Route::get('movies/{movie}', [MovieController::class, 'show'])->name('movies.show');
+
+Route::post('movies', [MovieController::class, 'store'])->name('movies.store');
+
+Route::put('movies/{movie}', [MovieController::class, 'update'])->name('movies.update');
+
+
+
+Route::get('delete', function () {
+    return view('movies.delete');
+});
+
+// Route::get(' ', function(){
+//     return view('home');
+// })->name('homepage');
+
 Route::get('/dashboard', function () {
     return view('dashboard');
 })->middleware(['auth'])->name('dashboard');
```

Comment:
This can also be moved into the group 😄

Reply:
'/dashboard' Route call has been moved into group